mods/mc_tutorial/init.lua

```diff
@@ -0,0 +1,1708 @@
+                    }
+                end,
+                collapsed_elem = function() 
+                    return {
+                        "label[0.4,3.1;Open the sidebar to modify keys!]"
+                    }
+                end,
+                expanded_elem = function()
+                    return {
+                        "button[0.4,2.9;3.4,0.8;key_add;Add selected]",
+                        "button[4,2.9;3.4,0.8;key_delete;Remove selected]",
+                    }
+                end,
+            },
+            [mc_tutorial.ACTION.LOOK_YAW] = {
+                name = "Look in horizontal direction (LOOK_YAW)",
+                fs_elem = function()
+                    return {
+                        "field[0.4,2.3;7,0.8;yaw;Yaw (horizontal degrees);", context.epop.fields.yaw or "", "]",
+                        "field_close_on_enter[yaw;false]",
+                    }
+                end,
+            },
+            [mc_tutorial.ACTION.LOOK_PITCH] = {
+                name = "Look in vertical direction (LOOK_PITCH)",
+                fs_elem = function()
+                    return {
+                        "field[0.4,2.3;7,0.8;pitch;Pitch (vertical degrees);", context.epop.fields.pitch or "", "]",
+                        "field_close_on_enter[pitch;false]",
+                    }
+                end,
+            },
+            [mc_tutorial.ACTION.LOOK_DIR] = {
+                name = "Look in direction (LOOK_DIR)",
+                fs_elem = function()
+                    local input_map = {
+                        [mc_tutorial.DIR.YAW_PITCH] = {
+                            name = "Yaw/pitch",
+                            get = function()
+                                local yaw, pitch
+                                if context.epop.fields.dir then
+                                    yaw, pitch = vect_to_yp(context.epop.fields.dir)
+                                end
+                                return table.concat({
+                                    "field[0.4,3.2;7,0.8;dir_yaw;Yaw (horizontal degrees);", yaw or "", "]",
+                                    "field[0.4,4.5;7,0.8;dir_pitch;Pitch (vertical degrees);", pitch or "", "]",
+                                    "field_close_on_enter[dir_yaw;false]",
+                                    "field_close_on_enter[dir_pitch;false]",
+                                })
+                            end,
+                        },
+                        [mc_tutorial.DIR.VECTOR] = {
+                            name = "Spatial vector",
+                            get = function()
+                                return table.concat({
+                                    "label[0.4,3.2;X =]",
+                                    "label[0.4,4.1;Y =]",
+                                    "label[0.4,5.0;Z =]",
+                                    "field[1,2.8;6.4,0.8;dir_x;;", context.epop.fields.dir and context.epop.fields.dir.x or "", "]",
```

Comment:
This is a spatial vector input, not a coordinate input.
Since it could be confusing to some, I could remove this input method altogether and restrict this to a yaw/pitch input, since that is simpler.

Reply:
Doh! This makes so much more sense. I think we can leave it in, but tool tips might be useful or a short description in the sidecar to the right. Maybe re-order the input options so that yaw/pitch is the default?